src/catalog/sources.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Sources
+---
+
+The Sources section lists the sources that contain stock for a specific product. It is used to modify the stock quantity for each product in a source. For more information about **Sources**, see [Managing Sources]({% link catalog/inventory-sources.md %}).
+
+![]({% link images/images/sources_view.png %}){: .zoom}
+_Sources View_
+
+It is possible to assign it to a source with a specific delivery method, such as an In-store delivery method.
```

Comment:
```suggestion
It is possible to assign it to a source with a specific delivery method, such as an in-store delivery method.
```

Reply:
true, does not make sense to have it here Kevin, I basically wanted to highlight that you can assign a source to a delivery method such as the in-store pickup delivery method